src/vuex/modules/session.js

```diff
@@ -261,11 +261,22 @@ export default ({ apollo }) => {
       })
       commit(`setUserAddressRole`, data.accountRole)
     },
-    getAllSessionsAddresses(store, { networkIds }) {
+    async getAllSessionAddresses({
+      rootState: {
+        session: { experimentalMode },
+      },
+    }) {
+      const { data } = await apollo.query({
```

Comment:
we already know the networks why requerying them?

Reply:
getting it from store. True... I thought of doing this in the beginning. Thanks